### ISSUE TITLE
[BUG] Blank Embeds with images

### ISSUE THREAD

**Issue by exor2008** [interactions-py/interactions.py#1428]
### Library Version

5.5.1

### Describe the Bug

Embed are added without a pictures. The picture starts to be displayed only after the message is deleted.

### Steps to Reproduce

1. Create Embed
2. Add a picture to embed
3. Call command with Embed
4. See a blank Embed
![image](https://github.com/interactions-py/interactions.py/assets/19377172/533d3713-c4d1-4249-8c2e-a679895b3ba0)

5. Delete message in discord chat
6. Call command with Embed again
7. Embed is displayed correctly
![image](https://github.com/interactions-py/interactions.py/assets/19377172/df5d34d1-ed31-438c-a05e-40975503933d)

### Expected Results

Expecting to see Embed with image immediately after publication
![image](https://github.com/interactions-py/interactions.py/assets/19377172/4b6adc25-62c2-4be5-972b-7924bf60d01b)

### Minimal Reproducible Code

```python
@slash_command("test", description="Test")
    async def test(self, ctx: SlashContext):
        embed = Embed()
        embed.add_image(
            "https://mobimg.b-cdn.net/v3/fetch/a1/a1b8ebc7037c04b79a8b3934b8799467.jpeg"
        )

        await ctx.respond(embed=embed)
```

### Checklist

- [x] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.
- [X] I have attempted to debug this myself, and I believe this issue is with the library

**Comment by AstreaTSS** [comment]
Hello - this is a Discord bug. Check https://github.com/discord/discord-api-docs/issues/6171 for information, and the latest post in the news channel in the interactions.py server for workarounds.

Also, this is a duplicate of #1419.

**Comment by exor2008** [comment]
Thanks for the quick response, @AstreaTSS 

Sorry for the duplicate post.

The problem is that the same bug did not reproduce on my discord.py version and it appeared exactly when it migrated to interactions.py, so I decided that the bug is on the library side. ¯\_(ツ)_/¯

**Comment by AstreaTSS** [comment]
This is likely due to differences in how d.py and i.py handle things (neither of which were/are wrong... it's weird). Again, check the Discord issue linked or the news post for more information.
